test/e2e/flask/multichain-api/wallet_invokeMethod.spec.ts

```diff
@@ -135,18 +132,13 @@ describe('Multichain API', function () {
               await driver.delay(largeDelayMs);
               await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
 
-              const accountWebElement = await driver.findElement(
-                '[data-testid="sender-address"]',
-              );
-              const accountText = await accountWebElement.getText();
               const expectedAccount =
                 i === INDEX_FOR_ALTERNATE_ACCOUNT ? 'Account 2' : 'Account 1';
 
-              assert.strictEqual(
-                accountText,
-                expectedAccount,
-                `Should have ${expectedAccount} selected, got ${accountText}`,
-              );
```

Comment:
this is an anti-pattern that leads to race conditions. We should look for the specific text
See more here https://github.com/MetaMask/metamask-extension/issues/19870